openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2496,6 +2617,7 @@ The technology described in this specification was made available from contribut
 
    -14
    
+   * add key attestation as additional information in a proof of possesion and new proof type
```

Comment:
leaving this here just we don't forget to move this description to the current newest draft in which this will end up - 15 right now